storage/innobase/dict/dict0dd.cc

```diff
@@ -7255,10 +7231,6 @@ bool dd_clear_encryption_flag(THD *thd, const char *space_name,
     dd_space->se_private_data().set(dd_space_key_strings[DD_SPACE_FLAGS],
                                     dd_space_flags);
     dd_space->options().set("encryption", "N");
-    if (clear_online_encryption) {
```

Comment:
@dutow, looks like  clear_online_encryption should be removed from this method

`/home/vsts/work/1/s/storage/innobase/dict/dict0dd.cc:7224:23: error: lambda capture 'clear_online_encryption' is not used [-Werror,-Wunused-lambda-capture]
  auto update_func = [clear_online_encryption](dd::Tablespace *dd_space) {
                      ^~~~~~~~~~~~~~~~~~~~~~~
1 error generated.
`

There may be some other Azure compile issues, haven't checked them all yet.